alpha/alpha-server/src/test/java/org/apache/servicecomb/pack/alpha/server/cluster/master/ClusterLockServiceTest.java

```diff
@@ -60,33 +63,105 @@
   ClusterLockService clusterLockService;
 
   @MockBean
-  private MasterLockRepository masterLockRepository;
+  private MasterLockEntityRepository masterLockEntityRepository;
 
-  @Test(timeout = 10000)
-  public void testClusterNodeType() throws InterruptedException {
-    //node type is master
-    clusterLockService.setMasterLock(null);
+  @Autowired
+  MasterLockRepository masterLockRepository;
+
+
+  @Test(timeout = 5000)
+  public void testMasterNode() throws InterruptedException {
+    MasterLock masterLock = clusterLockService.getMasterLock();
+    Assert.assertEquals(masterLock.getServiceName(),serviceName);
+    Assert.assertEquals(masterLock.getInstanceId(),instanceId);
+    Assert.assertEquals((masterLock.getExpireTime().getTime()-masterLock.getLockedTime().getTime()),expire);
+    when(masterLockEntityRepository.initLock(Mockito.anyString(),Mockito.any(Date.class),Mockito.any(Date.class),Mockito.anyString())).thenReturn(1);
+    when(masterLockEntityRepository.findMasterLockByServiceName(Mockito.anyString())).thenReturn(Optional.of(masterLock));
+    when(masterLockEntityRepository.updateLock(Mockito.anyString(),Mockito.any(Date.class),Mockito.any(Date.class),Mockito.anyString())).thenReturn(1);
+    Thread.sleep(2000);
```

Comment:
We can use waitUtil as we do in the accept test.

Reply:
OK, I will replace Mockito.any() with any()